lib/components/table/error.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import cn from 'classnames';
+import PropTypes from 'prop-types';
+import styles from './error.css';
+import AttentionIcon from '../icon/AttentionIcon';
+
+const VARIANT = {
+  ERROR: 'ERROR',
+  NO_DATA: 'NO_DATA',
+};
+
+const TableError = (props) => {
+  const { onClick, variant } = props;
+  let additionalWrapper = styles.noDataWrapper;
+  let icon;
+  let content = (
+    <>
+      <span data-testid="audit-try-again-msg">We didn’t find any Virtru activity.</span>
+      <span>
+        Encrypt or decrypt something and{' '}
+        <span data-testid="audit-try-again-btn" onClick={onClick} className={styles.underline}>
+          try again
+        </span>
+        .
+      </span>
+    </>
+  );
+
+  if (variant === VARIANT.ERROR) {
```

Comment:
This feels like the content sections should be pulled out into separate components and those components would pass the contents as `children` to the actual display component. So `TableError` would take the variant, delegate to the variant component which would pass `children` to `TableErrorDisplay`.

Probably not necessary yet though, so feel free to ignore.